mojaloop/values.yaml

```diff
@@ -8161,96 +11535,105 @@ mojaloop-bulk:
-      ##
-      enabled: false
-      ## @param tls.autoGenerated Generate a custom CA and self-signed certificates
-      ##
-      autoGenerated: true
-      ## @param tls.existingSecret Existing secret with TLS certificates (keys: `mongodb-ca-cert`, `mongodb-ca-key`, `client-pem`)
-      ## NOTE: When it's set it will disable certificate creation
-      ##
-      existingSecret: ""
-      ## Add Custom CA certificate
-      ## @param tls.caCert Custom CA certificated (base64 encoded)
-      ## @param tls.caKey CA certificate private key (base64 encoded)
-      ##
-      caCert: ""
-      caKey: ""
+      resources: {}
 
 ml-testing-toolkit:
   enabled: true
 
   ml-testing-toolkit-backend:
+
+    ## @param initContainers Add additional init containers to the %%MAIN_CONTAINER_NAME%% pod(s)
+    ## ref: https://kubernetes.io/docs/concepts/workloads/pods/init-containers/
+    ## e.g:
+    ## initContainers:
+    ##  - name: your-image-name
+    ##    image: your-image
+    ##    imagePullPolicy: Always
+    ##    command: ['sh', '-c', 'echo "hello world"']
+    ##
+    # initContainers: [] # We want to disable init-containers as there is no need
+    ## Use the following initContainers if you are using a MongoDB as a dependency
+    initContainers: |
+      - name: wait-for-mongodb
+        image: bitnami/mongodb:6.0.1
+        imagePullPolicy: IfNotPresent
+        command:
+          - sh
+          - -c
+          - |
+            until mongosh mongodb://${DB_HOST}:${DB_PORT} --authenticationDatabase "${DB_DATABASE}" --username "${DB_USER}" --password "${DB_PASS}" --eval "disableTelemetry();db.adminCommand('ping')" ;
+            do
+              echo --------------------;
+              echo Waiting for MongoDB...;
+              sleep 2; 
+            done;
+            echo ====================; 
+            echo MongoDB ok!;
+        env:
+          - name: DB_HOST
+            value: '{{ .Values.config.mongodb.host }}'
+          - name: DB_PORT
+            value: '{{ .Values.config.mongodb.port }}'
+          - name: DB_USER
+            value: '{{ .Values.config.mongodb.user }}'
+          - name: DB_PASS
+          {{- if not (empty .Values.config.mongodb.secret) }}
+            valueFrom:
+              secretKeyRef:
+                name: '{{ .Values.config.mongodb.secret.name }}'
+                key: '{{ .Values.config.mongodb.secret.key }}'
+          {{- else }}
+            value: {{ .Values.config.mongodb.password }}
+          {{- end }}
+          - name: DB_DATABASE
+            value: '{{ .Values.config.mongodb.database }}'
+
+    config:
+      mongodb:
+        host: *TTK_MONGO_HOST
+        port: *TTK_MONGO_PORT
+        user: *TTK_MONGO_USER
+        ## Secret-Management
+        ### Set this if you are using a clear password configured in the config section
+        password: *TTK_MONGO_PASSWORD
```

Comment:
```suggestion
        password: ''
```

Reply:
@elnyry-sam-k this is arelady set on the top-level config --> https://github.com/mojaloop/helm/blob/feat/%233082-utilise-externalised-secrets-for-user-password-management/mojaloop/values.yaml#L2